Problems with Numpy on Google Cloud

When running `make clean compile`, I get this error on Google Cloud:

```console
(venv) cs@instance-cs:~/wcEcoli$ make clean compile
rm -fr fixtures
(cd reconstruction/ecoli/dataclasses/process && rm -f equilibrium_odes.py two_component_system_odes*.py)
find . -name "*.pyc" -exec rm -rf {} \;
find . -name "*.o" -exec rm -fr {} \;
find . -name "*.so" -exec rm -fr {} \;
rm -fr build
rm -fr launcher_20* block_20*
python setup.py build_ext --inplace
Traceback (most recent call last):
  File "setup.py", line 9, in <module>
    import numpy as np
  File "/home/cs/wcEcoli/venv/local/lib/python2.7/site-packages/numpy/__init__.py", line 142, in <module>
    from . import add_newdocs
  File "/home/cs/wcEcoli/venv/local/lib/python2.7/site-packages/numpy/add_newdocs.py", line 13, in <module>
    from numpy.lib import add_newdoc
  File "/home/cs/wcEcoli/venv/local/lib/python2.7/site-packages/numpy/lib/__init__.py", line 8, in <module>
    from .type_check import *
  File "/home/cs/wcEcoli/venv/local/lib/python2.7/site-packages/numpy/lib/type_check.py", line 11, in <module>
    import numpy.core.numeric as _nx
  File "/home/cs/wcEcoli/venv/local/lib/python2.7/site-packages/numpy/core/__init__.py", line 26, in <module>
    raise ImportError(msg)
ImportError: 
Importing the multiarray numpy extension module failed.  Most
likely you are trying to import a failed build of numpy.
If you're working with a numpy git repo, try `git clean -xdf` (removes all
files not under version control).  Otherwise reinstall numpy.

Original error was: cannot import name multiarray

make: *** [Makefile:4: compile] Error 1
```

After this failure, I can't import numpy from a python shell. I can import numpy into a python shell again after uninstalling numpy and reinstalling, but it breaks again after running `make clean compile`. This is illustrated below:

```console
(venv) cs@instance-cs:~/wcEcoli$ pip uninstall numpy
DEPRECATION: Python 2.7 reached the end of its life on January 1st, 2020. Please upgrade your Python as Python 2.7 is no longer maintained. pip 21.0 will drop support for Python 2.7 in January 2021. More details about Python 2 support in pip, can be found at https://pip.pypa.io/en/latest/development/release-process/#python-2-support
Found existing installation: numpy 1.14.6
Uninstalling numpy-1.14.6:
  Would remove:
    /home/cs/wcEcoli/venv/bin/f2py
    /home/cs/wcEcoli/venv/lib/python2.7/site-packages/numpy-1.14.6.dist-info/*
    /home/cs/wcEcoli/venv/lib/python2.7/site-packages/numpy/*
Proceed (y/n)? y
  Successfully uninstalled numpy-1.14.6
(venv) cs@instance-cs:~/wcEcoli$ pip install numpy==1.14.6 --no-binary numpy --force-reinstall
DEPRECATION: Python 2.7 reached the end of its life on January 1st, 2020. Please upgrade your Python as Python 2.7 is no longer maintained. pip 21.0 will drop support for Python 2.7 in January 2021. More details about Python 2 support in pip, can be found at https://pip.pypa.io/en/latest/development/release-process/#python-2-support
Collecting numpy==1.14.6
  Using cached numpy-1.14.6.zip (4.9 MB)
Skipping wheel build for numpy, due to binaries being disabled for it.
Installing collected packages: numpy
    Running setup.py install for numpy ... done
Successfully installed numpy-1.14.6
(venv) cs@instance-cs:~/wcEcoli$ python
Python 2.7.16 (default, Oct 10 2019, 22:02:15) 
[GCC 8.3.0] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> import numpy
>>> exit()
(venv) cs@instance-cs:~/wcEcoli$ python
Python 2.7.16 (default, Oct 10 2019, 22:02:15) 
[GCC 8.3.0] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> import numpy
>>> exit()
(venv) cs@instance-cs:~/wcEcoli$ python
Python 2.7.16 (default, Oct 10 2019, 22:02:15) 
[GCC 8.3.0] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> import numpy as np
>>> np.mean([1, 2, 3])
2.0
>>> exit()
(venv) cs@instance-cs:~/wcEcoli$ python
Python 2.7.16 (default, Oct 10 2019, 22:02:15) 
[GCC 8.3.0] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> import numpy as np
>>> exit()
(venv) cs@instance-cs:~/wcEcoli$ make clean compile
rm -fr fixtures
(cd reconstruction/ecoli/dataclasses/process && rm -f equilibrium_odes.py two_component_system_odes*.py)
find . -name "*.pyc" -exec rm -rf {} \;
find . -name "*.o" -exec rm -fr {} \;
find . -name "*.so" -exec rm -fr {} \;
rm -fr build
rm -fr launcher_20* block_20*
python setup.py build_ext --inplace
Traceback (most recent call last):
  File "setup.py", line 9, in <module>
    import numpy as np
  File "/home/cs/wcEcoli/venv/local/lib/python2.7/site-packages/numpy/__init__.py", line 142, in <module>
    from . import add_newdocs
  File "/home/cs/wcEcoli/venv/local/lib/python2.7/site-packages/numpy/add_newdocs.py", line 13, in <module>
    from numpy.lib import add_newdoc
  File "/home/cs/wcEcoli/venv/local/lib/python2.7/site-packages/numpy/lib/__init__.py", line 8, in <module>
    from .type_check import *
  File "/home/cs/wcEcoli/venv/local/lib/python2.7/site-packages/numpy/lib/type_check.py", line 11, in <module>
    import numpy.core.numeric as _nx
  File "/home/cs/wcEcoli/venv/local/lib/python2.7/site-packages/numpy/core/__init__.py", line 26, in <module>
    raise ImportError(msg)
ImportError: 
Importing the multiarray numpy extension module failed.  Most
likely you are trying to import a failed build of numpy.
If you're working with a numpy git repo, try `git clean -xdf` (removes all
files not under version control).  Otherwise reinstall numpy.

Original error was: cannot import name multiarray

make: *** [Makefile:4: compile] Error 1
(venv) cs@instance-cs:~/wcEcoli$ python
Python 2.7.16 (default, Oct 10 2019, 22:02:15) 
[GCC 8.3.0] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> import numpy
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/cs/wcEcoli/venv/local/lib/python2.7/site-packages/numpy/__init__.py", line 142, in <module>
    from . import add_newdocs
  File "/home/cs/wcEcoli/venv/local/lib/python2.7/site-packages/numpy/add_newdocs.py", line 13, in <module>
    from numpy.lib import add_newdoc
  File "/home/cs/wcEcoli/venv/local/lib/python2.7/site-packages/numpy/lib/__init__.py", line 8, in <module>
    from .type_check import *
  File "/home/cs/wcEcoli/venv/local/lib/python2.7/site-packages/numpy/lib/type_check.py", line 11, in <module>
    import numpy.core.numeric as _nx
  File "/home/cs/wcEcoli/venv/local/lib/python2.7/site-packages/numpy/core/__init__.py", line 26, in <module>
    raise ImportError(msg)
ImportError: 
Importing the multiarray numpy extension module failed.  Most
likely you are trying to import a failed build of numpy.
If you're working with a numpy git repo, try `git clean -xdf` (removes all
files not under version control).  Otherwise reinstall numpy.

Original error was: cannot import name multiarray

>>> exit()
```

--- a/docs/create-pyenv.md
+++ b/docs/create-pyenv.md
@@ -19,6 +19,11 @@ This page goes through the Python environment setup steps in more detail and wit
 
 **Prerequisites:** Install the software tools as described in [dev-tools](dev-tools.md). That page covers installing pyenv and pyenv-virtualenv, initializing them in your shell profile, installing a C compiler, and more.
 
+**NOTE**: While it is possible to create a virtual environment with
+`virtualenv` or `venv` instead of using `pyenv`, make sure you store
+this environment *outside* the `wcEcoli/` root. Otherwise it will be
+wrecked when you run `make clean`.
+
 **Sherlock:** Sherlock is the Stanford scientific computing cluster. Outside the Covert lab, just skip our Sherlock notes. Inside the lab, look in `$PI_HOME/downloads/` and `$PI_HOME/installation_notes/` for downloaded software packages and notes on recompiling them as needed to install new packages or new versions for the team.
 
 
@@ -88,7 +93,7 @@ This page goes through the Python environment setup steps in more detail and wit
 1. Install Python using `pyenv`:
 
    ```bash
-   pyenv install 2.7.16 
+   pyenv install 2.7.16
    ```
 
 ## Create the "wcEcoli2" python virtual environment
@@ -133,7 +138,7 @@ This page goes through the Python environment setup steps in more detail and wit
    **Note:** If you get an `instruction not found` error while installing OpenBLAS, that probably means
    your old assembler is incompatible with the Fortran compiler. Figure out how to update the assembler
    or else install OpenBLAS 0.3.4 and suffer its threading bugs and inconsistent results.
-   
+
    **Note:** OpenBLAS 0.3.6+ works fine on macOS but not inside
    Docker on macOS unless you compile it with option `NO_AVX2=1`.
 

